tidy3d/components/lumped_element.py

```diff
@@ -146,54 +231,75 @@ def _voltage_axis_in_plane(cls, val, values):
             )
         return val
 
-    @cached_property
-    def _sheet_conductance(self):
+
+class LumpedResistor(RectangularLumpedElement):
+    """Class representing a rectangular lumped resistor. Lumped resistors are appended to the list
```

Comment:
Can you include working examples in the docstrings?

I think it would be helpful to even have complete examples showing the process from defining the lumped element through getting the complete simulation.